src/main/java/com/resources/Controller.java

```diff
@@ -1,72 +1,72 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 import twitter4j.Status;
 import twitter4j.TwitterException;
 
-import javax.ws.rs.*;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import java.util.Date;
 import java.util.List;
 
 
 @Produces(MediaType.APPLICATION_JSON)
-@Path("/api/1.0/twitter")
+@RestController
 public class Controller {
     public static final Logger logger = LoggerFactory.getLogger(Controller.class);
+    @Autowired
     TwitterImplement twitterImplement;
 
-
-    public Controller(TwitterImplement twitterImplement) {
-        this.twitterImplement = twitterImplement;
-    }
-
-    public Controller() {
-        twitterImplement = new TwitterImplement();
-    }
-
-    @GET
-    @Path("/healthCheck")
+    @RequestMapping("/healthCheck")
     public String healthCheck() {
         return "Ping Received at " + new Date();
     }
 
-    @GET
-    @Path("/getTweets")
-    public Response fetchTweets() {
-        return Response.ok(twitterImplement.getTweets()).build();
+    @RequestMapping(method =  RequestMethod.GET,value = "/getTweets")
+    public List<TwitterResponseModel> fetchTweets() {
+        return twitterImplement.getTweets();
     }
 
-    @GET
-    @Path("/filteredTweets")
+    @RequestMapping(method = RequestMethod.GET,value = "/filteredTweets")
     @Consumes(MediaType.APPLICATION_JSON)
-    public Response filteredTweets(@QueryParam("searchKey") String searchKey) throws TwitterException {
+    public List<TwitterResponseModel> filteredTweets(@QueryParam("searchKey") String searchKey) throws TwitterException {
         List<TwitterResponseModel> response = twitterImplement.getFilteredTweets(searchKey);
-        return Response.ok(response).build();
+        return response;
     }
 
-    @POST
-    @Path("/postTweets")
+    @RequestMapping(method = RequestMethod.GET ,value = "/tweetsPage")
     @Consumes(MediaType.APPLICATION_JSON)
-    public Response sendTweet(MessageRequest request) throws TwitterException {
+    public Response pagination(@QueryParam("start") int start, @QueryParam("size") int size) throws TwitterException {
+        List<TwitterResponseModel> response;
+        response = twitterImplement.getTweetsPage(start, size);
+        return Response.ok(response).build();
+    }
+
+    @RequestMapping(method = RequestMethod.POST, value = "/postTweets")
+    public SendResponse sendTweet(@RequestBody MessageRequest request) throws TwitterException {
         logger.info("got into post");
         String post = request.getMsg();
         if (StringUtil.isEmpty(post)) {
             logger.error("error happened");
-            return Response.status(400, "Please enter valid tweet").build();
+            return new SendResponse("Field is empty",400);
```

Comment:
Why did u changed the message? can keep it `Please enter valid tweet`

Reply:
Changed the message